### ISSUE TITLE
[BUG] Something was wrong! at fresh installation

### ISSUE THREAD

**Issue by JoseGonzalez84** [RedisInsight/Redis-for-VS-Code#213]
**Preconditions**

_VSCode_
- Versión: 1.94.2 (user setup)
- Confirmar: 384ff7382de624fb94dbaf6da11977bba1ecd427
- Fecha: 2024-10-09T16:08:44.566Z
- Electron: 30.5.1
- ElectronBuildId: 10262041
- Chromium: 124.0.6367.243
- Node.js: 20.16.0
- V8: 12.4.254.20-electron.0
- SO: Windows_NT x64 10.0.22631

_Redis for VS Code_
- v1.0.0

_Redis Insight_
- v2.58.0

**Steps to reproduce**
1)  Install Redis for VSCode from marketplace.
2)  Reload the window.
3) Open extension.
3) Error message will appear without any interaction

**Actual behavior** (A short description of what you found)
- This is a first time and fresh installation of Redis for VSCode.
- Connected with remote host with official extension.
- I've worked with Redis Insight and is working properly.

**Screenshots** (Paste or drag-and-drop a screenshot or a link to a recording)

I've attached a video with the explanation of failure.

https://github.com/user-attachments/assets/9cc2ddf8-4715-409d-bae8-4b2d63b0eb0f

_Extension output_
```
    2024-10-18T09:15:53.283Z - Extension activated
    2024-10-18T09:15:53.799Z - Current version: 1.0.0
    2024-10-18T09:15:53.837Z - Starting at port: 5542
    2024-10-18T09:16:01.844Z - BE started
```

**Comment by tsy-pavel** [comment]
This extension doesn't work. Previously worked with redis through another extension, everything works there. Decided to try yours and it shows that my database in redis is empty, but it's not.

**Comment by ViktarStarastsenka** [comment]
@tsy-pavel , can you share the results of the `INFO keyspace` command?
There is a chance that you do not have any keys in db0, which is a default database index that Redis for VS Code connects to.
Currently, it displays only one database index on UI (it will be enhanced with the subsequent releases). 
A database index can be specified on the form to add/edit a db connection under the "Select Logical Database" setting.

<img width="722" alt="image" src="https://github.com/user-attachments/assets/2cb8da49-329d-4db9-802d-3fca27bc2fdb">

**Comment by tsy-pavel** [comment]
Yes, if I add each index separately, it works. Thank you. But, of course, it would be much more convenient if it would show everything at once. Looking forward to the next releases)

**Comment by ViktarStarastsenka** [comment]
@tsy-pavel , support for database indexes has been added with 1.2.0 (released today).

**Comment by ViktarStarastsenka** [comment]
@JoseGonzalez84 , could you try it with 1.2.0?
please re-open the ticket if this issue persists.

**Comment by tsy-pavel** [comment]
Thank you, it is work very good 👍😊

**Comment by JoseGonzalez84** [comment]
> @JoseGonzalez84 , could you try it with 1.2.0? please re-open the ticket if this issue persists.
> 
> _updated_: if you use an SSH connection to VS Code, I think the issue will persist

Hi @ViktarStarastsenka , with the new version, this issue persist. There are no other changes in my environment.

**Comment by MellowOrchid** [comment]
I have the same problem as the author showed in the video, and we both installed VS Code on Windows and connected to a Linux machine through SSH. I hope this problem can be fixed in the next release, and I think this will be more convenient for developers.
By the way, there are some ERRORs in `~/.redis-for-vscode/logs/redisinsight-errors-*`:
```
2025/3/14 17:34:39 | ERROR | KeytarEncryptionStrategy | Failed to initialize keytar module | [object Object]
2025/3/14 17:34:40 | ERROR | TutorialsProvider | Unable to create static files from default | [object Object]
```

**Comment by ViktarStarastsenka** [comment]
@MellowOrchid , Redis for VS Code won’t launch in Remote Development mode because it includes binaries for the OS where it is installed. However, you can run Redis for VS Code locally and connect to your database via SSH.

**Comment by ViktarStarastsenka** [comment]
added this to the description of the current limitations together with a possible workaround - https://github.com/RedisInsight/Redis-for-VS-Code?tab=readme-ov-file#current-limitations

**Comment by JoseGonzalez84** [comment]
Sorry for insisting. But I think this extension has a significant limitation. Remote development is very common, and it's also common to develop across different operating systems.

The extension looks great and is officially supported by the Redis team, two very valuable points. However, this limitation negates this advantage. The advantage of using extensions is that they run in the same place as the code (like an IDE). Using a separate instance for this extension renders it useless. Instead, we can use Redis Insight or other free extensions that do the same thing.

Please consider this option for remote development. I think it would be very beneficial for this extension.

**Comment by ViktarStarastsenka** [comment]
@JoseGonzalez84 , can you delete the existing extension and cache (this is important), and then download a new one to ensure that it matches the operating system of your remote environment?
It should work.
This is not something we decide not to support - Redis for VS Code contains binaries for the OS on which it is installed.

**Comment by JoseGonzalez84** [comment]
For clean cache, do you refer this folders:

![Image](https://github.com/user-attachments/assets/c1e3394c-2da6-4c63-acf0-23368fb5e76e)

![Image](https://github.com/user-attachments/assets/94cfe601-07d4-4c17-b0ec-6492a1383de2)

![Image](https://github.com/user-attachments/assets/fb0200d0-bb93-4327-8525-1553da7c3db5)

Now, I open again VSCode, install new Redis for VSCode, I start again and when I configure a connection, the error persists.

(Please, note i'm using the option `Use SSH Tunnel`)

![Image](https://github.com/user-attachments/assets/15722c84-fabf-4738-887e-521156048736)

How can I download & install the extension for my remote environment OS? In my case, its Debian 12.

**Comment by ViktarStarastsenka** [comment]
@JoseGonzalez84 , you can try downloading a Linux package to install on your remote environment from here - https://open-vsx.org/extension/Redis/redis-for-vscode (with cache deleted beforehand).
If this doesn't work, I think the only way is to run the extension locally and then connect to your Redis server via SSH, without using Remote Development.

**Comment by JoseGonzalez84** [comment]
Well, thanks for your time.

Unfortunately, this didn't work either. I'll try again in the future.

Thanks for everything!
